src/TimePicker/TimePicker.js

```diff
@@ -180,7 +180,7 @@ class TimePicker extends React.Component {
   };
 
   isControlled() {
-    return this.props.value !== null;
+    return !!this.props.value;
```

Comment:
I feel like this line is totally fine. We use this very same logic on many component.
Whould setting `null` as a default value for `value` work?
(React might complain about a controlled && controlled behavior at the same time, but I'm not sure)